lib/Console/Command/Handler/ReportHandler.php

```diff
@@ -14,36 +14,58 @@
 
 use PhpBench\Model\SuiteCollection;
 use PhpBench\Report\ReportManager;
+use PhpBench\Storage\StorageRegistry;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class ReportHandler
 {
+    /**
+     * @var ReportManager
+     */
     private $reportManager;
 
-    public function __construct(ReportManager $reportManager)
+    /**
+     * @var StorageRegistry
+     */
+    private $storageRegistry;
+
+    /**
+     * @var SuiteCollectionHandler
+     */
+    private $suiteCollectionHandler;
+
+    public function __construct(ReportManager $reportManager, SuiteCollectionHandler $suiteCollectionHandler)
     {
         $this->reportManager = $reportManager;
+        $this->suiteCollectionHandler = $suiteCollectionHandler;
     }
 
-    public static function configure(Command $command)
+    public static function configure(Command $command): void
     {
         $command->addOption('report', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Report name or configuration in JSON format');
         $command->addOption('output', 'o', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'Specify output', ['console']);
+        SuiteCollectionHandler::configure($command);
     }
 
-    public function validateReportsFromInput(InputInterface $input)
+    public function validateReportsFromInput(InputInterface $input): void
     {
         $reportNames = $input->getOption('report');
         $this->reportManager->validateReportNames($reportNames);
     }
 
-    public function reportsFromInput(InputInterface $input, OutputInterface $output, SuiteCollection $collection)
+    public function reportsFromInput(InputInterface $input, OutputInterface $output, SuiteCollection $collection): void
     {
         $reports = $input->getOption('report');
         $outputs = $input->getOption('output');
+
+        if ($input->getOption('uuid')) {
```

Comment:
This could also work with `--file` and `--query` (even if `--query` will be removed)